setup.cfg

```diff
@@ -92,12 +92,6 @@ install_requires =
     cryptography>=0.9.3
     deprecated>=1.2.13
     dill>=0.2.2, <0.4
-    # Sphinx RTD theme 0.5.2. introduced limitation to docutils to account for some docutils markup
-    # change:
-    #      https://github.com/readthedocs/sphinx_rtd_theme/issues/1112
-    # This limitation can be removed after this issue is closed:
-    #      https://github.com/readthedocs/sphinx_rtd_theme/issues/1115
-    docutils<0.17
```

Comment:
I don't think this one caused the backtrack, but it's not needed anymore either way.

Reply:
I think the "lazy-object-proxy' in Dockerfile might be the reason  - see #20979

Reply:
I really wish this process was more predictable @uranusjr (but I can't think of any obvious way of making it more so)

Reply:
Agree. This ain't easy thing to do. But possibly making eager-update take into account the extras in "one pass" might be possible @uranusjr ? Not sure if that's possible, or if there are some reasons for that. I could possibly open an issue for that, but I am afraid my issues might not be too welcomed as my persistency in improving warning message in the `root` case is seen as a bit `toxic` (even if it is a real issue to tackle).

Reply:
Anyway, I from localy testing it appears that the backtrack is removed just by adding the restriction on importlib-metadata, and isn't anything to do with removing this docutils

Reply:
importlib-metadata was my next guess :)

Reply:
Either way, 3.7 image is pushing now so found and fixed the backtrack